### ISSUE TITLE
[med] : All Pages: Chrome + Jaws: Role is not defined for Delete workspace control.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8524]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to any workspace using Tab key and press using Enter to select it.
4. Navigate to More button using Tab key and press Enter to activate it.
5. Navigate to 'Delete workspace' control using Tab key.
6. Observe the screen reader announcement.

## Expected Result:
Role should be defined for the 'Delete Workspace' and when focus lands on the Delete button then screen reader should read "Delete Workspace button".

## Actual Result:
Focus lands on the 'Delete Workspace' control then screen reader is reading "Delete Workspace".

**Other occurrences**
>Same issue also repro on PR #8581, #8436, #8906,#8946, #9048, #9093 -Role is not defined for workspace and general setting.

## Workaround:
Yes

## Area issue was found in:
All pages of the application. 
Same issue repro on all controls present on the whole application and All pages.

## Failed WCAG checkpoints
4.1.2

## User impact:
This might confuse the user if the role is not announced. User will not be able to understand what action need to be perform on the control.

## Suggested resolution:
Provide Role = "button" for the  'Delete Workspace'.            
Refer to:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

## Platform:
- Web
- Desktop App
- Mobile Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti 

https://user-images.githubusercontent.com/97600950/162143288-fa46b25f-9fb4-44c9-97cf-36e8747e0ea1.mp4
